core/embed/rust/src/lib.rs

```diff
@@ -27,6 +32,12 @@ fn panic(_info: &PanicInfo) -> ! {
     // SAFETY: Safe because we are passing in \0-terminated literals.
     let empty = unsafe { CStr::from_bytes_with_nul_unchecked("\0".as_bytes()) };
     let msg = unsafe { CStr::from_bytes_with_nul_unchecked("rs\0".as_bytes()) };
+    // TODO: remove this debug print
+    use alloc::borrow::ToOwned;
+    let msg = format!("{}\0", info);
+    let msg = msg.to_owned();
+    let msg = &msg[..];
+    let msg = unsafe { CStr::from_bytes_with_nul_unchecked(msg.as_bytes()) };
```

Comment:
We should really handle this in some other way (i.e. enabling the panic handler only in production or something).

Reply:
Yeah. I let it up to you.